colleges/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+class Collage(models.Model):
+    name = CharField(unique=True)
+    address = TextField(blank=True)
+    phone_number = CharField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
```

Comment:
burada yazdığın class Meta yanlış. admin.ModelAdmin'in class Meta'sı. Model'inkini koymalısın.